_docs/tasks/basic-access-control.md

```diff
@@ -89,6 +101,25 @@ Using a whitelist is a two step process.
    `checkExpression` is evaluated and checked against the list `[v1, v2]`. The check behavior can be changed to a blacklist by specifying
    `blacklist: true`. The expression evaluator returns the value of the `version` label as specified by the `checkExpression` key.
 
+The alpha version of `istioctl` does not yet support
```

Comment:
Nit. We decided to stop using "alpha" throughout docs. We could just say the current version of istioctl.